templates/settings.py.erb

```diff
@@ -27,3 +27,12 @@ WORKING_DIRECTORY = "<%= scope['pulpcore::cache_dir'] %>"
 <% if scope['pulpcore::remote_user_environ_name'] -%>
 REMOTE_USER_ENVIRON_NAME = '<%= scope['pulpcore::remote_user_environ_name'] %>'
 <% end -%>
+AUTHENTICATION_BACKENDS = ['pulpcore.app.authentication.PulpNoCreateRemoteUserBackend']
+
+REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES = (
+    'rest_framework.authentication.SessionAuthentication',
+    'pulpcore.app.authentication.PulpRemoteUserAuthentication'
+)
+
+<%# This setting whitelists paths that can be used for repository sync with file protocol. Katello uses the path /var/lib/pulp/sync_imports/ to run tests%>
```

Comment:
IMHO it's fine to place this as a Python comment rather than an ERB comment. If you keep it as an ERB comment, please strip the trailing whitespace (`-%>` instead of `%>`).